keyboards/handwired/ergo_v_1/keyboard.json

```
@@ -0,0 +1,123 @@
+{
+    "keyboard_name": "Ergo-V-1",
+    "manufacturer": "wizardkeyboards",
+    "url": "https://wizardkeyboards.com/",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x97C4",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": false,
+        "command": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
```

Typically, bootmagic is preferred to be enabled.
```suggestion
        "bootmagic": true,
        "command": true,
        "extrakey": true,
        "mousekey": true,
```